test-guru/app/models/user.rb

```diff
@@ -3,7 +3,9 @@ class User < ApplicationRecord
   has_many :tests, through: :users_tests
   has_many :created_tests, class_name: "Test", foreign_key: "author_id", dependent: :nullify #при удалении пользователя тест, который создал пользователь, остается а информация об авторе обнуляется
 
+  validates :name, presence: true
   def passes_tests(level)
     self.tests.where(tests: { level: level})
```

Comment:
```suggestion
    tests.where(tests: { level: level})
```
"селф" можно не писать